ocaml/xapi/xha_interface.ml

```diff
@@ -180,7 +180,7 @@ module DaemonConfiguration = struct
     }
 
   let int_parameter (name, param) =
-    Opt.default [] (Opt.map (fun x -> [ xml_leaf_element name (string_of_int x) ]) param)
+    Option.value ~default:[] (Option.map (fun x -> [ xml_leaf_element name (string_of_int x) ]) param)
```

Comment:
can we replace this with something simpler?

```
match param with None -> [] | Some x -> [ xml_leaf_element name ... ]
```

if there are lots of statements like this, then maybe don't bother, but it's something to keep in mind

Reply:
Would `Option.fold` make this shorter? It would do the default value+map as one operation.